src/item.h

```diff
@@ -1925,7 +1932,10 @@ class item : public visitable<item>
          * PNULL.
          */
         phase_id current_phase = static_cast<phase_id>( 0 );
-
+        // The faction that owns this item.
+        const faction* owner = nullptr;
```

Comment:
I'd rather use faction_ids than raw pointers throughout this code.  But that can get cleaned up later when I scrap faction pointers in general.